README.md

```diff
@@ -201,6 +207,95 @@ const rule = {
 };
 ```
 
+### Includes
+
+The includes rule has three distinct behaviors, depending on whether the input is a string, an array, or an object.
+
+##### Input is a String
+
+When input is a string, the includes rule will check if the rule property in the config is a substring of the input.
+
+```yaml
+isSomeFeatureEnabled:
+  values:
+    - value: true
+      name: john
+    - value: false
+```
+
+```javascript
+const rule = {
+  type: 'includes',
+  input: input => input,
+  property: 'name',
+};
+...
+flipr.getValue(isSomeFeatureEnabled, 'john'); // true
+flipr.getValue(isSomeFeatureEnabled, 'johnathan'); // true
+```
+
+##### Input is an Array
+
+When input is an array, the includes rule will check if the rule property in the config exists in the input array.
+
+```yaml
+isSomeFeatureEnabled:
+  values:
+    - value: true
+      groups: admins
+    - value: false
+```
+
+```javascript
+const rule = {
+  type: 'includes',
+  input: input => input,
+  property: 'groups',
+};
+...
+flipr.getValue(isSomeFeatureEnabled, ['users', 'admins']); // true
+```
+
+##### Input is an Object
+
+When input is an object, the includes rule will check if the rule property in the config exists as a value in the input object.
+
+```yaml
+isSomeFeatureEnabled:
+  values:
+    - value: true
+      groups: admins
+    - value: false
+```
+
+```javascript
+const rule = {
+  type: 'includes',
+  input: input => input,
+  property: 'groups',
+};
+...
+flipr.getValue(isSomeFeatureEnabled, { groupA: 'users', groupB: 'admins' }); // true
+```
+
+### IncludesListAll
+
+The includesListAll rule is like the includes rule, in that it has the same three distinct behaviors, depending on whether the input is a string, an array, or an object. However, unlike the includes rules, the rule property in the config must always be an array. And **ALL** values in that array must match the includes logic.
+
+```yaml
+isSomeFeatureEnabled:
+  values:
+    - value: true
+      groups:
+        - admins
+        - superAdmins
+    - value: false
+```
+
+### IncludesListAny
+
+The includesListAny rule is like the includesListAll rule, except only one value in the rule property array must match the includes logic.
```

Comment:
`only one value` Shouldn't this be `at least one value`

Reply:
yup, thx